pkg/kinflate/transformers/noptransformer.go

```diff
@@ -0,0 +1,34 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package transformers
+
+import "k8s.io/kubectl/pkg/kinflate/types"
+
+// NOPTransformer contains a NOP transformer.
+type NOPTransformer struct{}
```

Comment:
I know naming is hard :)

Can we call it EmptyTransformer or NoOpTransformer to be explicit?